stormgate.proto

```diff
@@ -30,10 +30,20 @@ message ReplayChunk {
 }
 
 // 3 - Map identifier + RNG seed
-message Map {
-    string folder = 1;
-    string name = 2;
-    int32 seed = 3;
+message MapDetailsRecord {
+   string MapFolder = 1;
+   string MapName = 2;
+   int32 MapSeed = 3;   // Is different every lobby, so probably some kind of seed
+   int64 Unknown5 = 5;
+   int64 Unknown6 = 6;
+   MatchType MatchType = 7;
```

Comment:
Please stick with the convention of using `snake_case` for field names.

I also don't think there's any point in adding the Unknown fields.

Reply:
Was just Copy pasta CascadeFury's latest didn't even think about it